src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSHostImplementation.Types.cs

```diff
@@ -12,20 +12,32 @@ internal static partial class JSHostImplementation
 
         public sealed class PromiseHolder
         {
-            public nint GCVHandle;
+            public nint GCHandle;
```

Comment:
why is this nint instead of a GCHandle?

Reply:
Because sometimes it's a GCVHandle (virtual one). I will add comment.